core/primitives/src/errors.rs

```diff
@@ -626,9 +621,9 @@ impl Display for ActionErrorKind {
             ActionErrorKind::DeleteAccountStaking { account_id } => {
                 write!(f, "Account {:?} is staking and can not be deleted", account_id)
             }
-            ActionErrorKind::DeleteAccountHasRent { account_id, balance } => write!(
+            ActionErrorKind::DeleteAccountHasEnoughBalance { account_id, balance } => write!(
```

Comment:
Other people shouldn't be able to delete your account. Since you should never be able to go below your storage

Reply:
You can create your account without funds. What should we do in that case?

Reply:
You can't. The receipt will fail at the final check

Reply:
Let me add test for that.

Reply:
Added test and remove this error.